toolkit.lisp

```diff
@@ -13,6 +13,17 @@
 (defun unix->universal (unix)
   (+ unix *unix-epoch-difference*))
 
+(defun universal->timestring (universal)
+  (multiple-value-bind (seconds minutes hours day month year x daylight-saving timezone)
+      (decode-universal-time universal)
+    (declare (ignore x))
+    (let ((hour-offset (if daylight-saving
+                           (1- timezone)
+                           timezone)))
+    (format NIL
+            "~d-~2,'0d-~2,'0d ~2,'0d:~2,'0d:~2,'0dZ"
```

Comment:
I'm not sure what timestring is expected, but most likely it's an ISO8601 timestring, which requires the separator between date and time to be `T` rather than a space.